CartoDB/cartodb
# Use new loading buttons styles

Uses the new loading buttons in Builder as requested in #12132.

Example:
![loading_button](https://cloud.githubusercontent.com/assets/905225/26238114/f9085b74-3c77-11e7-98f3-fe242bc7fd49.gif)

Places where it has to be updated:
- [x] Run analysis.
- [x] Update the map.
- [x] Publish
- [x] Apply CSS.
- [x] Export as an image.
- [x] Apply SQL.

### How to test
Check the list above. In addition to this, check if we should update the analysis quota view as well.
